tests/e2e/specs/auth/gcp-flow-admin-2.test.js

```diff
@@ -44,6 +44,8 @@ describe( 'the set up flow for the second administrator', () => {
 				request.respond( { status: 200, body: '{}' } );
 			} else if ( url.match( 'user/data/survey-timeouts' ) ) {
 				request.respond( { status: 200, body: '[]' } );
+			} else if ( url.match( 'user/data/survey' ) ) {
+				request.respond( { status: 200, body: '{"survey":null}' } );
```

Comment:
This request should never be made because the site isn't using the proxy. This should be addressed by my earlier comment above.

Reply:
Reverted.